### PR TITLE
[feat] Setup Product Page layout and styles

### LINKED ISSUE #4: Setup Product Page layout and styles
For the Products page, lets setup the basic layout:

1. Setup the page so that filters are on the left, while other content take up almost 80% of the rightmost space
2.  Ensure alignment of all cards inside grid is functional

### PR DESCRIPTION
**Pull request by Hamdrive**
## Overview
This PR refers to issue #4 

This PR solely focused on the creation of the products page, including tis layout and styles, and any adaptions to React that were necessary.

Features/Changes implemented in this PR:

1. Setup static layout of Products page
2.  Update code of product site (filters, cards etc) to JSX
3. Creation of updated `Out Of Stock` card for better visibility
4. Porting of CSS styles into both CSS module and CSS utils file respectively.

## Preview
image

### CODE REVIEW

File: src/pages/products/Products.module.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+.overlay__text {
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  font-size: var(--size-17);
+  color: var(--color-white);
+}
+
+/* .card-brief {
```
Comment (saadahmadhere) [inline src/pages/products/Products.module.css]: it's good to remove unwanted comments

File: src/styles/utils.css [hunk 2]
```diff
@@ -236,5 +280,53 @@
...
+
+  .mob-aside {
+    position: relative;
+    min-height: 0;
+    height: fit-content;
+    padding-left: 0;
+    margin-top: 0;
+  }
+
+  /* .my-3 {
```
Comment (saadahmadhere) [inline src/styles/utils.css]: unwanted comments should be removed.

**Review by saadahmadhere — APPROVED** [review]
Well structured code. Nice use of variables. Sleek and clean.

File: src/pages/products/Products.module.css (hunk 1 above)
Comment (Hamdrive) [reply src/pages/products/Products.module.css]: Thank you for noticing! Removed.

File: src/styles/utils.css (hunk 2 above)
Comment (Hamdrive) [reply src/styles/utils.css]: Thank you for noticing! Removed.

**Review by shivampandey0 — APPROVED** [review]
Clean and Structured code with Components and Modular CSS.
Good to merge

**Comment by Hamdrive** [comment]
PR has been successfully reviewed by 2 members, thus satisfying the review criteria.
This fixes issue #4 and PR can be merged and closed.

**Pull request merged**